softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/interfaces/CommonRouterHandler.java

```diff
@@ -1,4 +1,4 @@
-package com.softgroup.common.router.api;
+package com.softgroup.common.router.api.interfaces;
```

Comment:
@werbower no need of special package interfaces. API package already mean than it's storing interfaces.